src/commands/git/worktree.ts

```diff
@@ -102,6 +102,8 @@ interface CreateState {
 		title?: string;
 	};
 
+	prId?: string;
```

Comment:
Where is this property being used? I see we are attaching `prId` to the "create" and "open" states but I'm having trouble finding where we use it. From what I can tell we only use it in the deep link service to open the multi-diff, but that is from the url itself, not the `prId` being set here.

Reply:
removed